tests/integrationtest/r/globalindex/mem_index_lookup.result

```diff
@@ -30,18 +66,54 @@ CREATE TABLE `t` (
 UNIQUE KEY `idx1` (`b`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin
 PARTITION BY HASH (`a`) PARTITIONS 5;
+insert into t values (2001, 2), (2002, 3), (2003, 4), (2004, 5);
 begin;
-insert into t values (2001, 2), (2002, 3), (2003, 4), (2004, 5), (2005, 1);
+insert into t values (2005, 1);
 explain select * from t use index(idx1) where b > 2;
 id	estRows	task	access object	operator info
-Projection_5	3333.33	root		globalindex__mem_index_lookup.t.a, globalindex__mem_index_lookup.t.b
-└─UnionScan_6	3333.33	root		gt(globalindex__mem_index_lookup.t.b, 2)
-  └─IndexLookUp_9	3333.33	root	partition:all	
-    ├─IndexRangeScan_7(Build)	3333.33	cop[tikv]	table:t, index:idx1(b)	range:(2,+inf], keep order:false, stats:pseudo
+Projection_5	3333.33	root	NULL	globalindex__mem_index_lookup.t.a, globalindex__mem_index_lookup.t.b
+└─UnionScan_6	3333.33	root	NULL	gt(globalindex__mem_index_lookup.t.b, 2)
+  └─IndexLookUp_11	3333.33	root	NULL	NULL
+    ├─Selection_10(Build)	3333.33	cop[tikv]	NULL	in(_tidb_pid, pid0, pid1, pid2, pid3, pid4)
```

Comment:
Will partition by hash(a) partitions 1024 display pid0,pid1,pid2, ... pid1023 ?
That would be terrible.

Reply:
I think so, do you have any better idea about it? Like don't show this `selection`?

Reply:
Let me write a doc tomorrow to ask PM and planner team.